### PR TITLE
Add observer position python interface

### PR DESCRIPTION
**Pull request by rhiannonlynne**
Issue #129

### CODE REVIEW

**Comment by Siltala** [comment]
Thanks for your contribution!
I've no problems with the code itself but I would suggest rewording the commit message to be more explicit ("Add observer position" is a little bit vague) on what it does, exactly, to something among the lines of "pyoorb: Implement oorb_observer", for example.

**Comment by rhiannonlynne** [comment]
Updated the commit message as suggested.

**Comment by Siltala** [comment]
Confirmed that this is also okay with @mgranvik so we're merging the PR. Thanks!
